source/options-storage.ts

```diff
@@ -19,6 +19,7 @@ const defaults = Object.assign({
 
 const migrations = [
 	featureWasRenamed('fix-view-file-link-in-pr', 'enable-file-links-in-compare-view'), // Merged on June 3rd
+	featureWasRenamed('filter-comments-by-you', 'discussion-filters'), // Merged on June 9th
```

Comment:
I'm being optimistic here :wink:

Reply:
We can drop this altogether since:

- it adds a new feture
- why would anyone disable an item in the dropdown :D 

```suggestion
```